handlers/Careers/components/CareersContent/styles.css

```diff
@@ -16,36 +15,67 @@
 }
 
 .CareersContent-item {
-  border-bottom: 1px solid #efefef;
-  padding: 1.5em;
+  box-sizing: border-box;
+  display: inline-block;
+  margin-bottom: 3em;
+  padding: 0 5%;
   position: relative;
-  list-style: none;
+  width: 50%;
 }
 
-.CareersContent-item:last-child {
-  border-bottom: 0;
+.CareersContent-item-segment {
+  border-bottom: 1px solid #efefef;
+  max-width: 382px;
+  padding-bottom: 2em;
+}
+
+.CareersContent-button {
+  margin-top: 1.4em;
+}
+
+.CareersContent-button:hover {
```

Comment:
let's use postCSS and add a mixin that adds ```:focus``` everywhere there is a ```:hover```, because I always forget!